src/worker.js

```diff
@@ -33,6 +42,21 @@ function start() {
     logger.info(`starting ${job}`);
     await reconcileLegacyReports();
   });
+
+  // Notifications
+  notificationQueue.on('failed', (job, error) => auditLogger
+    .error(`job ${job.name} failed for report ${job.data.report.displayId} with error ${error}`));
+  notificationQueue.on('completed', (job, result) => {
+    if (result != null) {
+      logger.info(`Succesfully sent ${job.name} notification for ${job.data.report.displayId}`);
+    } else {
+      logger.info(`Did not send ${job.name} notification for ${job.data.report.displayId} because SEND_NOTIFICATIONS is not set`);
```

Comment:
Nice that we have a logged record. 👍  Question - would the flag being set to `false `be the only reason the job would not complete here?

Reply:
If it errors, it is handled differently. If the result is null, that is because it skipped sending the notification because the flag was set to false.